images/macos/provision/core/xcode-postbuild.sh

```diff
@@ -15,6 +15,10 @@ do
     #add sleep to let CoreSimulatorService to exit
     sleep 3
 
+    # Version 12.2_beta installed into 12.2 directory
+    if [[ $XCODE_VERSION == "12.2_beta" ]] ; then
+        XCODE_VERSION="12.2"
```

Comment:
It is not workaround but expected behavior for all future Xcode versions. I suggest replacing this logic to just `XCODE_VERSION=$(echo $XCODE_VERSION | cut -d"_" -f 1)`

Reply:
cool, didn't know that, thanks!